stories/SQFormDropdown.stories.tsx

```diff
@@ -1,10 +1,28 @@
 import React from 'react';
+import type {AnySchema} from 'yup'; 
 
 import {SQFormDropdown as SQFormDropdownComponent} from '../src';
+import type {SQFormDropdownProps} from 'components/SQForm/SQFormDropdown';
 import {SQFormStoryWrapper} from './components/SQFormStoryWrapper';
-import {createDocsPage} from './utils/createDocsPage';
+import type {SQFormStoryWrapperProps} from './components/SQFormStoryWrapper';
 import getSizeProp from './utils/getSizeProp';
-import markdown from '../notes/SQFormDropdown.md';
+import type {gridOptions} from './utils/getSizeProp';
+import type { Meta, Story } from '@storybook/react';
+import {createDocsPage} from './utils/createDocsPage';
+import * as markdown from '../notes/SQFormDropdown.md';
+
+type FormProps = {
+  initalValues?: SQFormStoryWrapperProps['initialValues'];
+} & Omit<SQFormStoryWrapperProps, 'initalValues' | 'children'>;
+
+type DropdownStoryType = Story<
+  Omit<SQFormDropdownProps, 'size'> &
+  {
+   size?: gridOptions,
+   SQFormProps?: FormProps,
```

Comment:
How come `SQFormProps` is capitalized?
Generally in React, the only time a Prop is capitalized is when the Prop is a Component.

ex:

```typescript
type Props = {
  IconComponent: React.ReactNode // or whatever
}
```

Reply:
Yeah not sure why it was that way. I guess I just followed suit. I will clean it up.